### ISSUE TITLE
文字描述&关键内容修订建议

### ISSUE THREAD

**Issue by fandywang** [nndl/nndl.github.io#19]
- [x] 1. ch1 page1 “要通过真正地通过图灵测试” -> "要真正地通过图灵测试"
- [ ] 2. ch1 page2 "由大量的节点（或称“神经元”，或“单元”）和之间相互联接构成" -> "由大量的节点（或称“神经元”，或“单元”）之间相互联接构成"
- [ ] 3. ch1 page2 "甚至在很多的任务上比算法重要" -> “甚至在很多的任务上比算法更重要”
- [ ] 4. ch2 page9 "一个标量c与矩阵A乘积为A的每个元素是A的相应元素与c的乘积" -> "一个标量c与矩阵A乘积为cA的每个元素是A的相应元素与c的乘积"
- [ ] 5. ch2 page10 "一个n × n的对角矩阵矩阵A满足" -> "一个n × n的对角矩阵A满足"
- [ ] 6. ch2 page10 "n × n的对角矩阵矩阵A" -> "n × n的对角矩阵A"
- [ ] 7. ch2 2.1 建议增加下特征值/特征向量，可逆矩阵，行列式，矩阵求导，常见基本初等函数求导示例等内容（有些内容在2.5中，建议调整到2.1）
- [ ] 8. ch2 page12 2.24等式右边两项颠倒下顺序？
- [ ] 9. ch2 page12 2.25式 tr 在前面并未介绍
- [ ] 10. ch2 2.2章节 “数学优化” -> “数值优化”?尤其在第二段刚开始描述是“数值优化问题的定义为”，后面同样也出现了数值优化说法，建议统一成更为常见的数值优化叫法
- [ ] 11. 对 R 和 Z （可能还有其他符号）并未说明，也许需要有一个字符集说明？
- [ ] 12. ch2 page13 “离散优化问题的求解一般都是比较困难，优化算法的复杂度都比较高。” -> “离散优化问题的求解一般都比较困难，优化算法的复杂度都比较高。” 
- [ ] 13. ch2 page13 整数规划部分也仿照组合优化部分给一些典型的例子？
- [ ] 14. ch2 page14 “有约束条件的约束问题常常可以通过拉格朗日乘数转化为非约束问题。” -> “有约束条件的约束优化问题常常可以通过拉格朗日乘子法转化为无约束优化问题。” （or 拉格朗日乘数法）
- [ ] 15. ch2 page15 “所对于所有的x” -> "对于所有的 x"
- [ ] 16. ch2 page15 定理2.2 二阶导表达式错误
- [ ] 17. ch2 page16 “有一阶必要性定理可知∇f(x)则” -> "有一阶必要性定理可知∇f(x)=0，则"
- [ ] 18. ch2 page16 “如果顺利的话序列，(xn)收敛到局部最优解x∗。” -> “如果顺利的话序列(xn)收敛到局部最优解x∗。”
- [ ] 19. ch2 page17 如果可以对牛顿法多展开介绍下就更好了，甚至涵盖 BFGS，LBFGS，OWLQN 等优化方法。另外，共轭梯度法前面提了一句，是否也加一些内容？
- [ ] 20. ch2 2.3章节中“实验”和“试验”说法统一下吧
- [ ] 21. ch2 page18 “随机事件（或简称事件）指的是一个被赋与机率的事物集合” -> “随机事件（或简称事件）指的是一个被赋予概率的事物集合”
- [ ] 22. ch2 page18 “概率表示对一个随机事件发生的可能性大小” -> "概率表示一个随机事件发生的可能性大小"
- [ ] 23. ch2 page19 随机变量 X 的符号都用斜体吧
- [ ] 24. ch2 page21 “若随机变量X 服从一个位置参数为µ和σ”->“若随机变量X 服从一个未知参数为µ和σ”
- [ ] 25. ch2 page23 多项分布定义部分重复给出了二项分布定义，应该没必要，删了吧，直接到“把二项分布推广到随机向量，就得到了多项分布。假设一个袋子中装了很 XXXX”
- [ ] 26. ch2 page23 式2.57 p 改为 θ？
- [ ] 27. ch2 page23 式2.58 gamma 函数 Γ(x) 的定义给下？比如在右侧补充下？
- [ ] 28. ch2 page23 “从这种表示形式和Dirichlet分布类似” -> "这种表示形式和Dirichlet分布类似"
- [ ] 29. ch2 page23 既然最后提到 Dirichlet分布，不妨也在合适的位置介绍下？
- [ ] 30. ch2 2.4章节很多本来是下标，但没有正确呈现出来，如 xi，xj
- [ ] 31. ch2 page26 I(X)  是什么意思，要给出解释吧？
- [ ] 32. ch2 page27 2.4.2章节英文描述翻译成中文吧？
- [ ] 33. ch2 page27 2.4.2章节DKL(p∥q) 要给出解释吧？
- [ ] 34. ch2 page29 "在1844-1845年研究种群数量的增长模型时提出的命名的"->"在1844-1845年研究种群数量的增长模型时提出命名的"
- [ ] 35. ch2 page 29 σ(x) 第一次使用时，并未明确说明就是 logistic 函数的表示方法，反而是在其下面解释的：一般使用标准logistic函数（记为σ(x)）
- [ ] 36. ch2 page29 式2.90是空的
- [ ] 37. ch3 page33 “有监督学习” 使用粗体吧，与下面的 item 风格保持一致
- [ ] 38. ch3 page33 3.1.1章节可以提下序列标注？
- [ ] 39. ch3 page34 “降低对标记数据数量的要求”->"降低对标注数据数量的要求"
- [ ] 40. ch3 page35 “yi 也也可以看成是真实类别的分布”->“yi 也可以看成是真实类别的分布”
- [ ] 41. ch3 page36 过拟合的定义中两个 h 没有区分开
- [ ] 42. ch3 page36 “为了解决过拟合问题，一般在经验风险最小化的原则上上加参数的正则化”->“为了解决过拟合问题，一般在经验风险最小化的原则上加上参数的正则化”
- [ ] 43. ch3 page36 式3.14 L2范数不需要加上标
- [ ] 44. ch3 page37 如果能增加更多 L1和 L2范数的原理和对比介绍就更好了
- [ ] 45. ch3 page37 在前文中“样本”还被描述成“实例”，也许可以统一叫法，或者在“样本”介绍时说明下
- [ ] 46. ch3 page38 “数据的原始表示转换为。” 转换成什么呢？
- [ ] 47. ch3 page39 "在机器学习，搜索步长α中也叫作学习率（Learning Rate）。"->"在机器学习中，搜索步长α也叫作学习率（Learning Rate）。"
- [ ] 48. ch3 page40 "Mini-Batach"->"Mini-Batch"
- [ ] 49. ch3 page40 Mini-Batch 兼顾随机和批量方法优点，但是前文并未提及批量方法优点是什么。另外，SGD 方法的不足也可以提下，这样说 Mini-Batch 是一种折中方法逻辑更清晰
- [ ] 50. ch3 page40 线性搜索和自适应学习率如果能有更详细介绍，或者给出参考资料更好
- [ ] 51. ch3 page41~42 可以介绍下为什么 动量法、adagrad、adadelta有效，以及适用什么场景？
- [ ] 52. ch3 page42 RMSprop、Adam等也许也可以提下？
- [ ] 53. ch3 page43、47、50 “给定N 给样本”->"给定N 个样本"
- [ ] 54. ch3 page46 式3.47 分子少了右括号
- [ ] 55. ch3 page47 式3.54 为空
- [ ] 56. ch3 page48 式3.59 使用 λ 表示学习率，而前文中一直是 a，最好统一下？
- [ ] 57. ch3 page48 “空间中的存在一些区域” 不通顺
- [ ] 58. ch3 page48 多分类通过二分类实现及存在缺陷，如果通过直观的图呈现出来更佳
- [ ] 59. ch3 page49 “对比公式3.65中的两类分类判别函数” 这里的公式3.65标错了吧
- [ ] 60. ch3 page49 softmax 回归最后是否提一句和最大熵模型推到出来的后验概率形式一致
- [ ] 61. ch3 page50 “采样梯度下降法”->"采用梯度下降法"
- [ ] 62. ch3 page51 “其中，I()为指示函数。” 频繁的出现，也许没必要?
- [ ] 63. ch3 page52 "决策树模型可以读性好，具有描述性"->"决策树模型可读性好，具有描述性"
- [ ] 64. ch3 page53 式3.82漏掉了 p(c) ，先验概率还是不能省略的，除非显式说明各类别出现概率均等
- [ ] 65. ch3 page54 公式 3.83 给的太直接了，很难理解，且缺少一些解释
- [ ] 66. ch3 page56 “正确率和召回率是广泛用于信息检索和统计学分类领域的两个度量”-> "准确率和召回率是广泛用于信息检索和统计学分类领域的两个度量" 
- [ ] 67. ch3 page56 “F1 值是根据正确率和召回率二者给出的一个综合的评价指标”->“F1 值是根据准确率和召回率二者给出的一个综合的评价指标"
- [ ] 68. ch3 page58 “1960年代,,”-> "1960年代,"
- [ ] 69. ch3 page57 3.4 章节可以考虑加下奥卡姆剃刀原理？
- [ ] 70. ch4 页面标号没有和 ch3 连上
- [ ] 71. ch4 page49 “感知器也可以看出是线性分类器的一个经典学习算法。”->"感知器也可以看做是线性分类器的一个经典学习算法。" 
- [ ] 72. ch4 page51 公式4.3，"w∗Txi <0 当yi <0." -> "w∗Txi <=0 当yi <0." 
- [ ] 73. ch4 page51 "具体的学习过程如算法5.2所示。"->"具体的学习过程如算法4.1所示。"
- [ ] 74. ch4 page51 算法4.1 第6行多了一个逗号
- [ ] 75. ch4 page52 “具体的学习过程如算法5.2所示。”->“具体的学习过程如算法4.1所示。”
- [ ] 76. ch4 page53 “如果训练集不 s 是线性可分的” ->"如果训练集不是线性可分的"
- [ ] 77. ch4 page53 "算法5.2"->"算法4.1"
- [ ] 78. ch4 page54 “比如输出是序列或来其它结构化的形式。”-> "比如输出是序列或者其它结构化的形式。"
- [ ] 79. ch4 page57 公式4.15第二行少了一个左括号
- [ ] 80. ch4 page58 算法4.3 第8、9行少了φ
- [ ] 81. ch5 page75 公式5.13 未写完整
- [ ] 
- [ ] 未完待续

**Comment by xpqiu** [comment]
感谢指出错误！如果方便的话可否告诉我你的名字，我在书中正式致谢：）

**Comment by fandywang** [comment]
谢谢邱老师大作，我叫王莉峰
